config/config.go

```diff
@@ -8,28 +8,30 @@ import (
-	FeedbackEnabled            bool          `envconfig:"FEEDBACK_ENABLED"`
-	SearchControllerURL        string        `envconfig:"SEARCH_CONTROLLER_URL"`
-	SearchRoutesEnabled        bool          `envconfig:"SEARCH_ROUTES_ENABLED"`
-	APIRouterURL               string        `envconfig:"API_ROUTER_URL"`
-	DownloaderURL              string        `envconfig:"DOWNLOADER_URL"`
-	PatternLibraryAssetsPath   string        `envconfig:"PATTERN_LIBRARY_ASSETS_PATH"`
-	SiteDomain                 string        `envconfig:"SITE_DOMAIN"`
-	RedirectSecret             string        `envconfig:"REDIRECT_SECRET" json:"-"`
-	SQSAnalyticsURL            string        `envconfig:"SQS_ANALYTICS_URL"`
-	ContentTypeByteLimit       int           `envconfig:"CONTENT_TYPE_BYTE_LIMIT"`
-	HealthckeckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
-	HealthckeckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	BindAddr                            string        `envconfig:"BIND_ADDR"`
+	BabbageURL                          string        `envconfig:"BABBAGE_URL"`
+	RendererURL                         string        `envconfig:"RENDERER_URL"`
+	CookiesControllerURL                string        `envconfig:"COOKIES_CONTROLLER_URL"`
+	HomepageControllerURL               string        `envconfig:"HOMEPAGE_CONTROLLER_URL"`
+	DatasetControllerURL                string        `envconfig:"DATASET_CONTROLLER_URL"`
+	FilterDatasetControllerURL          string        `envconfig:"FILTER_DATASET_CONTROLLER_URL"`
+	GeographyControllerURL              string        `envconfig:"GEOGRAPHY_CONTROLLER_URL"`
+	GeographyEnabled                    bool          `envconfig:"GEOGRAPHY_ENABLED"`
+	FeedbackControllerURL               string        `envconfig:"FEEDBACK_CONTROLLER_URL"`
+	FeedbackEnabled                     bool          `envconfig:"FEEDBACK_ENABLED"`
+	SearchControllerURL                 string        `envconfig:"SEARCH_CONTROLLER_URL"`
+	SearchRoutesEnabled                 bool          `envconfig:"SEARCH_ROUTES_ENABLED"`
+	APIRouterURL                        string        `envconfig:"API_ROUTER_URL"`
+	DownloaderURL                       string        `envconfig:"DOWNLOADER_URL"`
+	PatternLibraryAssetsPath            string        `envconfig:"PATTERN_LIBRARY_ASSETS_PATH"`
+	SiteDomain                          string        `envconfig:"SITE_DOMAIN"`
+	RedirectSecret                      string        `envconfig:"REDIRECT_SECRET" json:"-"`
+	SQSAnalyticsURL                     string        `envconfig:"SQS_ANALYTICS_URL"`
+	ContentTypeByteLimit                int           `envconfig:"CONTENT_TYPE_BYTE_LIMIT"`
+	HealthckeckCriticalTimeout          time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	HealthckeckInterval                 time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	ZebedeeRequestMaximumTimeoutSeconds time.Duration `envconfig:"ZEBEDEE_REQUEST_TIMEOUT_SECONDS"`
```

Comment:
Is this too verbose, is it necessary to prepend env config with SECONDS?

Reply:
It's worth updating the README.md with configurations too

Reply:
So the zebedee client in dp-api-client.go `New` function calls it: `ZEBEDEE_REQUEST_TIMEOUT_SECONDS` so I opted for continuity, although not convinced we have any secrets with that config in play. (but imagine it was added at some point as something surly most have needed it, or else why add it).

Reply:
Also readme has been updated now, good catch